HW7.py

```diff
@@ -0,0 +1,44 @@
+phone_book = {
+    'Igor': 132434,
+    'Diana': 254345,
+}
+
+print (phone_book.keys())
+
+while True:
+    user_input = input('Enter a command: ')
+    split_input = user_input.split()
+   
+    command = split_input[0]
+    name = split_input[1]
+    number = split_input[2]
+ 
+    if phone_book.get(name) is None:  #тут у мене щось не виходить - як зробити, щоб можно було ввести лише command?
```

Comment:
I don't understand what this part is related to. What do you wand to check here?

Reply:
the question is no longer relevant